GherkinSyncTool/GherkinSyncTool/Synchronizers/TestRailSynchronizer/TestRailSynchronizer.cs

```diff
@@ -36,7 +38,9 @@ public void Sync(List<IFeatureFile> featureFiles)
                 foreach (var scenario in featureFile.Document.Feature.Children.OfType<Scenario>())
                 {
                     var tagId = scenario.Tags.FirstOrDefault(tag => Regex.Match(tag.Name, config.TagIdPattern, RegexOptions.IgnoreCase).Success);
-
+                    if(_config.TestRailMaxRequestsPerMinute is not null)
```

Comment:
Let's move this in the `TestRailClientWrapper `as well?

Reply:
You mean put it inside RequestLimitCheck method?